test_createlisting_sql_inj.py::Security_Of_SQL_Injection_Tes::test_test_createlisting
--------------------------------------------------------------------
Last Page: http://127.0.0.1:8080/login
 Duration: 8.37s
  Browser: Chrome 100.0.4896.60
   Driver: chromedriver 100.0.4896.60
Timestamp: 1668822984  (Unix Timestamp)
     Date: Friday, November 18, 2022
     Time: 8:56:24 PM  (EDT, UTC-05:00)
--------------------------------------------------------------------
Traceback: File "/Library/Frameworks/Python.framework/Versions/3.10/lib/python3.10/unittest/case.py", line 591, in run
    self._callTestMethod(testMethod)
  File "/Library/Frameworks/Python.framework/Versions/3.10/lib/python3.10/unittest/case.py", line 549, in _callTestMethod
    method()
  File "/Users/yuanqiliang/Desktop/CISC327/qbay_test/test_security/test_createlisting_sql_inj.py", line 34, in test_test_createlisting
    self.click("a[href='/create'")
  File "/Library/Frameworks/Python.framework/Versions/3.10/lib/python3.10/site-packages/seleniumbase/fixtures/base_case.py", line 287, in click
    element = page_actions.wait_for_element_visible(
  File "/Library/Frameworks/Python.framework/Versions/3.10/lib/python3.10/site-packages/seleniumbase/fixtures/page_actions.py", line 426, in wait_for_element_visible
    timeout_exception(NoSuchElementException, message)
  File "/Library/Frameworks/Python.framework/Versions/3.10/lib/python3.10/site-packages/seleniumbase/fixtures/page_actions.py", line 189, in timeout_exception
    raise exc(msg)
Exception: Message: 
 Element {a[href='/create'} was not present after 7 seconds!
